.github/workflows/build-and-run-model.yaml

```diff
@@ -65,6 +66,9 @@ jobs:
       # Maximum pipeline runtime. This is slightly below 6 hours, which
       # is the maximum length of any single GitHub Actions job
       role-duration-seconds: 21000
+      # Disable Batch job status polling since this workflow often takes
+      # more than 6 hours
+      poll_for_status: false
```

Comment:
We could also think about making this a workflow input variable if we think we might want some jobs to poll for status and some not to, but for now I went with the simplest path forward.